src/composables/authLogIn.js

```diff
@@ -1,23 +1,27 @@
 import { signInWithEmailAndPassword } from "firebase/auth";
-import { firebaseAuth } from "@/firebase/config";
+import { firebaseAuth, firebaseDb } from "@/firebase/config";
 import { useState } from "react";
+import useAuthContext from "@/context/useAuthContext";
 
 function useAuthLogIn() {
+  const { dispatch } = useAuthContext();
 
-    const [addUser, setAddUser] = useState("");
-    const [firebaseError, setFirebaseError] = useState("")
+  const [firebaseError, setFirebaseError] = useState("");
 
-    const LogIn = ({  email, password }) => {
-      signInWithEmailAndPassword(firebaseAuth, email, password)
-          .then((userCredential) => {
-            setAddUser(userCredential.user)
-            //  Handle successful login
-          })
-          .catch((error) => {
-            setFirebaseError(error.message)
-        });
-    };
-    return { addUser, firebaseError, LogIn };
+  const LogIn = ({ email, password }) => {
+    signInWithEmailAndPassword(firebaseAuth, email, password)
+      .then(async (userCredential) => {
+        // setAddUser(userCredential.user)
+        const docRef = doc(firebaseDb, "MANAGERS", userCredential?.user?.uid);
+        const docSnap = await getDoc(docRef);
+        dispatch({ type: "SIGN_IN", payload: docSnap.data() });
+        //  Handle successful login
```

Comment:
@ibimina Do we actually handle the successful login? When I login to the local app, my login credentials print to console but I am not redirected to `managerDashboard`

Reply:
After a successful login the user should be redirected to the managerDashboard page

Reply:
@msarit I refactored the login function, The login function was not working because it was not hooked up with firebase